contrib/opentracing/interceptor.go

```diff
@@ -164,9 +165,19 @@ type tracerSpanRef struct{ opentracing.SpanContext }
 type tracerSpan struct{ opentracing.Span }
 
 func (t *tracerSpan) Finish(opts *interceptor.TracerFinishSpanOptions) {
-	if opts.Error != nil {
+	if opts.Error != nil && !isBenignApplicationError(opts.Error) {
```

Comment:
Open tracing is [deprecated](https://www.cncf.io/blog/2022/01/31/cncf-archives-the-opentracing-project/), not sure we should bother fixing it here/ do a release to publish this